### PR TITLE
py-xdg: new port (version 6.0.0)

### PR DESCRIPTION
**Pull request by catap**
#### Description

I've also renamed old port `py-xdg` into `py-pyxdg` which is correct name, updated all dependencies and fixed build of `py-bpython`.

I've also fix build py-bpython by modern setuptoools

###### Type(s)

- [x] bugfix
- [x] enhancement

###### Tested on

macOS 12.6.8 21G725 x86_64
Xcode 14.2 14C18

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-bpython/Portfile [hunk 1]
```diff
@@ -25,7 +25,12 @@ checksums               rmd160  6a57d45732b0f756668a481c99e8c3135c741e67 \
 python.versions         27 37 38 39 310
 
 if {${name} ne ${subport}} {
-    livecheck.type          none
+    post-patch {
+        # fix version detection
+        set versionpath ${worksrcpath}/bpython/_version.py
+        copy ${filespath}/_version.py ${versionpath}
+        reinplace "s/__VERSION__/${version}/" ${versionpath}
+    }
```
Comment (lpsinger) [inline python/py-bpython/Portfile:33]: This doesn't look related to the changes described in the PR.

File: python/py-xdg/Portfile [hunk 2]
```diff
@@ -1,37 +1,27 @@
...
+version                 6.0.0
+revision                0
+
+supported_archs         noarch
+platforms               {darwin any}
+license                 ISC
+maintainers             {@catap korins.ky:kirill} {@RhetTbull gmail.com:rturnbull} openmaintainer
+
+description             Variables defined by the XDG Base Directory Specification
+long_description        {*}${description}
```
Comment (lpsinger) [inline python/py-xdg/Portfile:16]: What does the `{*}` do?

File: python/py-xdg/Portfile [hunk 3]
```diff
@@ -1,37 +1,27 @@
...
+
+homepage                https://github.com/srstevenson/xdg-base-dirs
```
Comment (lpsinger) [inline python/py-xdg/Portfile:18]: Consider using the github port group.

File: python/py-pyxdg/Portfile [hunk 4]
```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-pyxdg
+version             0.28
+license             LGPL-2
+categories-append   textproc
+maintainers         {aronnax @lpsinger} openmaintainer
```
Comment (lpsinger) [inline python/py-pyxdg/Portfile:10]: Please feel free to add yourself as a maintainer.

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: python/py-bpython/Portfile (hunk 1 above)
Comment (catap) [reply python/py-bpython/Portfile:33]: @lpsinger without this changes this port simple doesn't build anymore.

Comment (catap) [reply python/py-bpython/Portfile:33]: If you would like I may keep it as dedicated commit.

File: python/py-xdg/Portfile (hunk 2 above)
Comment (catap) [reply python/py-xdg/Portfile:16]: @lpsinger value of variable as multiple arguments, and not just one argument to that function.
```
description             Variables defined by the XDG Base Directory Specification
long_description        ${description}
long_description        {*}${description}
```
Here tree lines:
 1. calls `description` with arguments: `Variables`, `defined`, `by`, and so on;
 2. calls `long_description` with one argument which is value which is returned by `${description}`;
 3. calls `long_description` with multiple arguments similar to (1)

File: python/py-xdg/Portfile (hunk 3 above)
Comment (catap) [reply python/py-xdg/Portfile:18]: I think that using PyPi to get pythons artifacts is cleaner and I'm using it when this option is available.

But you're question points me that I must add `xdg-base-dirs` to keep synchronized. Thanks!

File: python/py-pyxdg/Portfile (hunk 4 above)
Comment (catap) [reply python/py-pyxdg/Portfile:10]: ok

File: python/py-bpython/Portfile (hunk 1 above)
Comment (catap) [reply python/py-bpython/Portfile:33]: BTW I've kept it as dedicated commit but haven't mentored it inside PR description. This was changed.

Comment (lpsinger) [reply python/py-bpython/Portfile:33]: I don't have enough information here to review this change. How did the build fail? How does this fix it?

File: python/py-xdg/Portfile (hunk 2 above)
Comment (lpsinger) [reply python/py-xdg/Portfile:17]: OK, I don't know enough Tcl myself to verify that.

**Comment by catap** [comment]
@lpsinger am I right that I’ve addressed all your questions?

**Comment by catap** [comment]
@mascguy can it be merged?

**Review by mascguy — APPROVED** [review]

**Comment by catap** [comment]
Thanks!
